### ISSUE TITLE
New `secrets` hook

### ISSUE THREAD

**Issue by jhunt** [starkandwayne/genesis#257]
Build a secrets hook that gets executed for genesis secret management.  Possible invocations:

```
./hooks/secrets add <env> <prefix>
./hooks/secrets check <env> <prefix>
./hooks/secrets rotate <env> <prefix>
```

These will currently only be responsible for rotating in the Vault, not credhub.

**Comment by dennisjbell** [comment]
👍

**Comment by jhunt** [comment]
This has been implemented int he v2.6 line.
